.gitignore

```diff
@@ -5,3 +5,4 @@ test.js
 docker_data/
 docs/
 browser/bcoin*
+.idea
```

Comment:
What's this for?

Reply:
This can be removed. It's to keep my IDE (Webstorm) state files from being tracked by git.

Reply:
finally went ahead and added .idea to my global ignore